content/applications/finance/fiscal_localizations/austria.rst

```diff
@@ -0,0 +1,137 @@
+=======
+Austria
+=======
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the following modules to get all the features of the Austrian
+localization.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+     - License
+   * - :guilabel:`Austria - Accounting`
+     - `l10n_at`
+     - Default :ref:`fiscal localization package <fiscal_localizations/packages>`.
+     - Community
+   * - :guilabel:`Austria - Accounting Reports`
+     - `l10n_at_reports`
+     - Adds localized versions of financial reports
+     - Enterprise
+   * - :guilabel:`Austrian SAF-T Export`
+     - `l10n_at_saft`
+     - Adds the SAF-T export.
+     - Enterprise
+
+.. image:: austria/austria-modules.png
+   :alt: Modules for the Austrian localization
+
+Financial Reports
+=================
+
+.. seealso::
+   :doc:`../accounting/reporting`
+
+Currently the following localized reports are available:
+
+  - Balance sheet according to § 224 UGB
+  - Profit and loss according to § 231 UGB (Gesamtkostenverfahren)
+
+To populate the individual lines of these reports the mapping from
+your company's chart of accounts to the SAF-T accounts is used.
```

Comment:
The sentence is not so clear, not sure the good meaning is conveyed. Try maybe to put it in active form starting with "The mapping from your company's chart of accounts.."